spec/models/ingredient_recipe.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe IngredientsRecipes, type: :model do
+
+  subject do
+    build(:ingredient_recipe)
+  end
+
+  describe 'when observing relationships' do
+    context 'with recipes' do
+      it 'belongs to an ingredient' do
+        ingredient = Ingredient.new(name: 'Test Ingredient')
```

Comment:
We shouldn't be using Model.new here. Remember to use the rspec create and build syntax. This applies all over these tests

Reply:
okay